### PR TITLE
Me: E-Mail subscription: Rephrase Digests description

### PR DESCRIPTION
**Pull request by akirk**
Currently the _Digests_ e-mail subscription is explained with the description _Reading & Writing digests, tailored for you_. While this is very much to the point, one must know that these mean:
- Reader digest: you might like to read these things that others published. you are a reader.
- Writer digest: here’s a look at what you published and how it did in the last month. you are a writer.

I'd like to suggest to change the description to something easier to understand, for example _Your personal feed with interesting tidbits to read and digests about your writing._

Other suggestions are welcome!

cc @meremagee 

screencapture at thu oct 6 21 48 58 edt 2016

### CODE REVIEW

**Comment by lancewillett** [comment]
> I'd like to suggest to change the description to something easier to understand, for example Your personal feed with interesting tidbits to read and digests about your writing.

How about:

"A compilation of interesting tidbits from both your personal feed and your own writing."

@akirk

**Comment by akirk** [comment]
I like it, thanks @lancewillett! I've updated the PR.

**Comment by akirk** [comment]
@meremagee Can you agree with this change?

**Comment by umurkontaci** [comment]
@akirk Looks like it's been a while :) Are you still willing to follow through with this? If so, can you rebase and ask for another review?

**Comment by akirk** [comment]
@gwwar Thanks for digging this up. Rebased and added the label.

**Comment by michelleweber** [comment]
Overall comment on this screen: we should always spell out "and" unless the & is part of the actual name of something. So, research and surveys, online and in-person, etc.

For this particular line, I'd get a little more specific: "Popular content from the blogs you follow, and reports on your own site and its performance."

File: client/me/notification-settings/wpcom-settings/index.jsx [hunk 1]
```diff
@@ -86,39 +87,40 @@ const WPCOMNotifications = React.createClass( {
...
+					title={ this.props.translate( 'Research' ) }
+					description={ this.props.translate( 'Opportunities to participate in WordPress.com research and surveys.' ) }
 				/>
 
 				<EmailCategory
 					name={ options.community } isEnabled={ this.state.settings.get( options.community ) }
-					title={ this.translate( 'Community' ) }
-					description={ this.translate( 'Information on WordPress.com courses and events (online & in-person).' ) }
+					title={ this.props.translate( 'Community' ) }
+					description={ this.props.translate( 'Information on WordPress.com courses and events (online and in-person).' ) }
```
Comment (a8ci18n) [inline client/me/notification-settings/wpcom-settings/index.jsx]: :disappointed: 29 existing translations will be lost with this change.

File: client/me/notification-settings/wpcom-settings/index.jsx [hunk 2]
```diff
@@ -86,39 +87,40 @@ const WPCOMNotifications = React.createClass( {
...
+					description={ this.props.translate( 'Tips for getting the most out of WordPress.com.' ) }
 				/>
 
 				<EmailCategory
 					name={ options.research }
 					isEnabled={ this.state.settings.get( options.research ) }
-					title={ this.translate( 'Research' ) }
-					description={ this.translate( 'Opportunities to participate in WordPress.com research & surveys.' ) }
+					title={ this.props.translate( 'Research' ) }
+					description={ this.props.translate( 'Opportunities to participate in WordPress.com research and surveys.' ) }
```
Comment (a8ci18n) [inline client/me/notification-settings/wpcom-settings/index.jsx]: :disappointed: 30 existing translations will be lost with this change.

File: client/me/notification-settings/wpcom-settings/index.jsx (hunk 1 above)
Comment (akirk) [inline client/me/notification-settings/wpcom-settings/index.jsx]: I'll take care of them not getting lost :)

File: client/me/notification-settings/wpcom-settings/index.jsx (hunk 2 above)
Comment (akirk) [inline client/me/notification-settings/wpcom-settings/index.jsx]: I'll take care of them not getting lost :)

**Comment by akirk** [comment]
Thanks, I have used your suggestion. Unfortunately the patch is now bigger because I needed to update the translate calls.

**Comment by rachelmcr** [comment]
@akirk Ready to merge?

**Pull request merged**